website/client/src/components/ui/filterDropdown.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <span>
     <span class="dropdown-label">{{ label }}</span>
-    <b-dropdown right="right">
+    <b-dropdown
+    :text="(selectedItem.id.includes('sortBy')) ? selectedItem.id.substring(6) : selectedItem.id"
+    right="right">
       <span
         slot="text"
```

Comment:
@prosarahgamal I would try to change `slot="text"` with `slot="button-content"` here and remove your other changes

Reply:
@paglias Thanks a lot for your valuable comments
It works fine now